packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -451,7 +454,20 @@ impl PostgresDatabase {
                 .change_context(InsertionError)?;
         }
 
-        // TODO: Store link references
+        // FIXME: Without `collect` we get a weird lifetime error
+        let entity_type_references = entity_type
+            .inner()
+            .link_references()
```

Comment:
Now that we've gotten over the naming confusion, can we be explicit about calling link-types link types. And here you're actually looking at linked_entity_types or something else along those lines.

Reply:
I think this is fine. For this snippet
```json
"links": {
  "https://blockprotocol.org/@alice/types/link-type/written-by/v/1": {
    "$ref": "https://blockprotocol.org/@alice/types/entity-type/person/v/1"
  },
  "https://blockprotocol.org/@alice/types/link-type/contains/v/1": {
    "type": "array",
    "items": {
      "$ref": "https://hash.ai/@hash/types/entity-type/block/v/1"
    }
  }
}
```
`link_references` will return a map:
```rust
[
    (
        "https://blockprotocol.org/@alice/types/link-type/written-by/v/1", 
        "https://blockprotocol.org/@alice/types/entity-type/person/v/1"
    ),
    (
        "https://blockprotocol.org/@alice/types/link-type/contains/v/1",
        "https://hash.ai/@hash/types/entity-type/block/v/1"
    ),
]
```

Reply:
I see. Can leave it as is for this PR, I'm going to be touching this code in my next one anyway, can rename if needed there